### PR TITLE
Remove lifetime parameter of Renderer

### PR DESCRIPTION
**Pull request by sotaroikeda**
In #2032, @glennw commented about the problem about lifetime parameter of Renderer. The lifetime parameter could cause a problem to merge to Servo.  This pull request addresses it. There could be another ways to address the problem.

cc @glennw @kvark

---
This change is [Reviewable](https://reviewable.io/reviews/servo/webrender/2064)

### CODE REVIEW

File: wrench/src/rawtest.rs [hunk 1]
```diff
@@ -8,18 +8,18 @@ use std::sync::mpsc::Receiver;
...
 
-impl<'a, 'b> RawtestHarness<'a, 'b> {
-    pub fn new(wrench: &'a mut Wrench<'b>, window: &'a mut WindowWrapper, rx: Receiver<()>) -> Self {
+impl<'a> RawtestHarness<'a> {
+    pub fn new(wrench: &'a mut Wrench, window: &'a mut WindowWrapper, rx: Receiver<()>) -> Self {
         RawtestHarness {
-            wrench,
-            rx,
-            window,
+            wrench: wrench,
```
Comment (kvark) [inline wrench/src/rawtest.rs]: this should be unnecessary

**Comment by glennw** [comment]
Thanks! Looks good to me once the nit from @kvark above is fixed.

**Comment by sotaroikeda** [comment]
Apply the comment!

**Comment by glennw** [comment]
Thanks!

@bors-servo r+

**Pull request merged**